File: datasophon-common/src/main/java/com/datasophon/common/model/Host.java
package com.datasophon.common.model;

import lombok.Data;
import lombok.Getter;

@Data
@Getter
public class Host {
    
    private Long projectEnvDetailId;
    
    private String ip;
    
    private Integer port;
    
    private String user;
    
    private String password;
    
    private String hostname;
    
}

File: datasophon-cli-go/internal/config/host.go
package config

// Host 对应 Java Host model。
type Host struct {
	ProjectEnvDetailID int64  `yaml:"projectEnvDetailId"`
	IP                 string `yaml:"ip"`
	Port               int    `yaml:"port"`
	User               string `yaml:"user"`
	Password           string `yaml:"password"`
	Hostname           string `yaml:"hostname"`
}
